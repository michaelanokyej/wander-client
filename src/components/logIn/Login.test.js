import React from "react";
import Login from "./Login"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"

describe(`Login component`, () => {
    it(`renders the Login component`, () => {
        const wrapper = shallow(<Login />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});