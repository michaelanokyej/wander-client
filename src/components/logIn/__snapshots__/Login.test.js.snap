// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component renders the Login component 1`] = `
<main>
  <header
    role="banner"
  >
    <h2>
      Welcome Back!
    </h2>
    <h4>
      Please log in
    </h4>
  </header>
  <form
    className="signup-form"
  >
    <div>
      <label
        htmlFor="username"
      >
        Email
      </label>
      <input
        id="username"
        name="username"
        onChange={[Function]}
        type="email"
      />
    </div>
    <div>
      <label
        htmlFor="password"
      >
        Password
      </label>
      <input
        id="password"
        name="password"
        onChange={[Function]}
        type="password"
      />
    </div>
    <button
      onClick={[Function]}
      type="submit"
    >
      Log in
    </button>
  </form>
</main>
`;
